### ISSUE TITLE
Enhance `skore.cross_validate`: specify which metric is actually used behind `test_score`

### ISSUE THREAD

**Issue by sylvaincom** [probabl-ai/skore#578]
### Problem

Part of epic https://github.com/probabl-ai/skore/issues/544

When using `skore.cross_validate` with linear regression for example, I can see the following scores in the legend: `test_score` and `r2_score`, while the `test_score` is actually the `r2_score`! This can be confusing for the user.

### Suggested solution

It is ok the keep `test_score` and `r2_score`, but in any case, tell the user which score is actually used by default by scikit-learn for the `test_score` (it depends on each estimator).

**Comment by augustebaum** [comment]
So would we show this information in the plot? Or does it also have to be in the output of `cross_validate` itself (given the constraint that we should try to stick to scikit-learn's output format)?

**Comment by augustebaum** [comment]
How exactly should we give this information? In the title of the plot?

**Comment by sylvaincom** [comment]
Hmm subtitle of the plot?

**Comment by tuscland** [comment]
@sylvaincom the parent issue is closed, but this one is still open. That reveals a workflow issue, can you please check?

**Comment by sylvaincom** [comment]
We need to address this, mentioned today in https://github.com/probabl-ai/skore/issues/1047

**Comment by sylvaincom** [comment]
In the case of the regression task, let's just display `R2 (⭐️)` and `RMSE` where the star indicates that it's scikit-learn's default

WDYT? Cc @augustebaum @MarieS-WiMLDS

**Comment by MarieS-WiMLDS** [comment]
I like the star idea because it's pretty, but is it technically possible to have an emoji in the legend? Plus, it is explicit? The other less pretty option but explicit and easy would be to print sth like ` R2 (sklearn default)`. You can do tests with Auguste and see how it seems :) !

**Comment by sylvaincom** [comment]
After discussing with @glemaitre, this issue is on hold, waiting on #997 and #1014 as they will change the way a cross-validation report is displayed

**Comment by sylvaincom** [comment]
Since v0.6.1, we have the `EstimatorReport` and the `CrossValidationReport`. This issue is outdated and superseded by https://github.com/probabl-ai/skore/issues/1217.
